project/backend/prometheus.yml

```
@@ -0,0 +1,46 @@
+  - job_name: 'project-manager'
+    metrics_path: /actuator/prometheus
+    static_configs:
+      # Target uses the container name defined in project-manager/docker-compose.yml
+      - targets: ['project-manager-app:9087'] # App internal port
+
+  # Job to scrape API Gateway App (Assuming Spring Boot Actuator)
+  - job_name: 'api-gateway'
+    metrics_path: /actuator/prometheus
+    static_configs:
+      # Target uses the container name defined in api-gateway/docker-compose.yml
+      - targets: ['api-gateway-app:9000'] # App internal port
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)